vendor/src/github.com/docker/libnetwork/drivers/overlay/joinleave.go

```diff
@@ -49,33 +49,33 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 
 	sbox := n.sandbox()
 
-	name1, name2, err := createVethPair()
+	overlayIfName, containerIfName, err := createVethPair()
 	if err != nil {
 		return err
 	}
 
-	ep.ifName = name2
+	ep.ifName = overlayIfName
```

Comment:
Yes, that's correct, actually it makes no difference, either interfaces would be fine.
But during code-review in libnetwork we decided the overlay driver to hold the name of the interface on which the driver operates. Just to make it more intuitive.